pkg/cmd/issue/shared/lookup.go

```diff
@@ -142,6 +221,37 @@ type PartialLoadError struct {
 	error
 }
 
+func FindIssuesOrPRs(httpClient *http.Client, repo ghrepo.Interface, issueNumbers []int, fields []string) ([]*api.Issue, error) {
```

Comment:
This is the content of `IssuesFromArgWithFields` without the parse step, so it operates directly on issue numbers and an already determined repo.